docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md

```diff
@@ -12,76 +12,46 @@ Follow these sections to create the Gauge dashboard type for the `cpu_temperatur
-
-   ![Temperature](./assets/temperature-celsius.png)
+2. Find the **Field** tab and set the measuring unit to Celsius degrees to reflect the metric data type.
 
 3. Set the minimum metric value to `60` and the maximum value to `90` to reflect the `cpu_temperature_celsius` metric value range.
 
-   ![Minimum and maximum values](./assets/min-max-values.png)
-
-4. Set a red color threshold to `75` for the dashboard to turn red once the CPU temperature reaches and exceeds this value.
-
-   ![Threshold](./assets/threshold.png)
-
-5. Go to the **Panel** tab and give a title to the dashboard.
+4. For the dashboard to turn red once the CPU temperature reaches and exceeds 75 °C, set a red color threshold to `75`.
 
-   ![Panel title](./assets/panel-title.png)
+5. Go to the **Panel** tab and give a title to the dashboard - for example, `CPU Temperature`.
 
-6. Enable the **Labels** option to display this range on the dashboard.
+6. Under **Panel > Display**, make sure the threshold labels and threshold markers are activated to display this range on the dashboard.
 
-   ![Panel title](./assets/enable-labels.png)
-
-7. Click the save button in the top right corner of the page to save the changes. Add an optional note to describe the changes made.
-
-   ![Note](./assets/save-note.png)
+7. Save your changes. We recommend that you add a note to describe the changes made.
 
 ## Verify the dashboard
 
 Refresh the browser to see how the dashboard changes according to the current value of the `cpu_temperature_celsius` metric.
 
-- It turns green if the current metric value ranges from 60 to 74 degrees Celsius:
-
-   ![Green dashboard](./assets/green-dashboard.png)
-
-- It turns red if the current metric value ranges from 75 to 90 degrees Celsius:
+- It turns **green** if the current metric value ranges from 60 to 74 degrees Celsius:
 
-   ![Red dashboard](./assets/red-dashboard.png)
+- It turns **red** if the current metric value ranges from 75 to 90 degrees Celsius:
 
-You can also define the dashboard's ConfigMap and add it to the `resources` folder under the given component's chart. To make the dashboard visible, simply use the `kubectl apply` command to deploy it. For details on adding monitoring to components, see the [`README.md`](https://github.com/kyma-project/kyma/blob/master/resources/monitoring/charts/grafana/README.md) document.
+## Next steps
 
-> **NOTE:** If you don't want to proceed with the following tutorial, [clean up the configuration](obsv-05-clean-up-configuration.md).
+- Follow the tutorial to [Define alerting rules](obsv-04-define-alerting-rules-monitor.md).
+- You can also define the dashboard's ConfigMap and add it to the `resources` folder under the given component's chart. To make the dashboard visible, simply use the `kubectl apply` command to deploy it. For details on adding monitoring to components, see the [`README.md`](https://github.com/kyma-project/kyma/blob/master/resources/monitoring/charts/grafana/README.md) document.
```

Comment:
```suggestion
- You can also define the dashboard's ConfigMap and add it to the `resources` folder under the given component's chart. To make the dashboard visible, simply use the `kubectl apply` command to deploy it. For details on adding monitoring to components, see the [Grafana chart `README.md`](https://github.com/kyma-project/kyma/blob/master/resources/monitoring/charts/grafana/README.md).
```
I feel like just `README` is not descriptive enough. 
I'd phrase it `see the Grafana chart README` or as in the suggestion above.

Reply:
fully agree.